customers/tests/conftest.py

```diff
@@ -80,6 +82,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def mocked_response_my_profile(data="", *args, **kwargs):
+    def wrapper(*args, **kwargs):
+        if data != "":
```

Comment:
?
```suggestion
        if data:
```

Reply:
I left the default as `""` because if the customer profile is not created, the data will be `None` and I wanted it so that I would be able to test that case.

Reply:
Added comment in code to clarify